collection-spec/collection-spec.md

```diff
@@ -97,7 +97,106 @@ The following types are commonly used as `rel` types in the Link Object of a Col
 
 | Type    | Description                                                  |
 | ------- | ------------------------------------------------------------ |
-| self    | **REQUIRED.** *Absolute* URL to the collection. This is required, to represent the location that the file can be found online. This is particularly useful when in a download package that includes metadata, so that the downstream user can know where the data has come from. |
+| self    | # STAC Collection Specification
+
+The STAC Collection Specification defines a set of common fields to describe a group of Items that share properties and metadata. The 
```

Comment:
Here's something wrong, text got duplicated.

Reply:
Yes, noticed that also and fixed right after.